portal/client/src/components/ListView/ListView.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const ListView = ({studyList = [], handleClick}) => {
+    return (
+        <div className="col-4">
```

Comment:
Lets keep this stuctural classes in parent one, so, that this component can just focus on displaying the things and can be reused, without adding any complicated structural classes